CONTRIBUTING.md

```diff
@@ -0,0 +1,187 @@
+# Contributing
+
+This document provides guidelines for contributing to samples.
+
+## Dependencies
+
+The following dependencies must be installed on the development system:
+
+- [Docker Engine][docker-engine]
+- [Google Cloud SDK][google-cloud-sdk]
+- [make]
+
+## Linting and Formatting
+Files in the repository are linted or formatted to maintain a standard of quality and statically validated.
+You can run this check locally:
+
+```
```

Comment:
Can we add `cd terraform-docs-samples` here?

Reply:
Done